eth_utils/abi.py

```diff
@@ -66,3 +78,49 @@ def event_signature_to_log_topic(event_signature: str) -> bytes:
 def event_abi_to_log_topic(event_abi: Dict[str, Any]) -> bytes:
     event_signature = _abi_to_signature(event_abi)
     return event_signature_to_log_topic(event_signature)
+
+
+def get_normalized_abi_component_type(abi_component: Union[Dict[str, Any], str]) -> str:
+    """
```

Comment:
I don't see these docs get auto-generated. We should either fix that or move this blurb to `docs/utilities.py`